backend/src/main/java/com/redhat/cloud/notifications/auth/kessel/KesselAuthorization.java

```diff
@@ -210,10 +214,14 @@ protected LookupResourcesRequest buildLookupResourcesRequest(final RhIdentity id
                     .setSubject(
                         ObjectReference.newBuilder()
                             .setType(ObjectType.newBuilder().setNamespace(KESSEL_RBAC_NAMESPACE).setName(KESSEL_IDENTITY_SUBJECT_TYPE).build())
-                            .setId(identity.getName())
+                            .setId(getUserId(identity))
                     ).build()
             ).setRelation(kesselPermission.getKesselPermissionName())
             .setResourceType(ResourceType.INTEGRATION.getKesselObjectType())
             .build();
     }
+
+    private String getUserId(RhIdentity identity) {
+        return backendConfig.getKesselDomain() + "/" + identity.getUserId();
```

Comment:
The suggested format from the Kessel team used the colon instead of the forward slash character.

```suggestion
        return backendConfig.getKesselDomain() + ":" + identity.getUserId();
```

Reply:
I confirmed with Alec, the latest version should use a '/'. As he said, it changed a few times :)